### ISSUE TITLE
False positive in a compute property with tuple

### ISSUE THREAD

**Issue by rabc** [nicklockwood/SwiftFormat#347]
Hi,

It might be related to #346. SwiftFormat is removing the return keyword in the following case:

```
var myVar: (x: CGFloat, y: CGFloat) {
    return (0.0, 1.0)
}
```

All other computed properties are fine, it is just when the return type is a tuple.

**Comment by nicklockwood** [comment]
@rabc thanks for reporting, I'll get it fixed asap. In the meantime, the same workarounds suggested in 
https://github.com/nicklockwood/SwiftFormat/issues/346 should apply.

**Comment by nicklockwood** [comment]
@rabc fixed in 0.37.5.
